kitsune/sumo/middleware.py

```diff
@@ -53,7 +53,7 @@ class HttpResponseRateLimited(HttpResponse):
 
 class SUMORefreshIDTokenAdminMiddleware(SessionRefresh):
     def __init__(self, *args, **kwargs):
-        if not settings.OIDC_ENABLE:
+        if not settings.OIDC_ENABLE or settings.DEV:
```

Comment:
Maybe it would be less error prone if you add a config entry for the refresh middleware specifically.

Reply:
The OIDC_ENABLE is already used and I would prefer not to alter this. The only other use case that we have is to disable the oidc only for the admin interface for dev purposes.